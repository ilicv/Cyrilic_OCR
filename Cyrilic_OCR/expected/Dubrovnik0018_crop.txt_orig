слѝнгурина ж (пеј. од слѝнгура) – 
Јѐдна му је ћи̑ у̀шесна, а дру̏га̄ пра̑ва̄
слѝнгурина.

сли̏ндра̄н, -а, -о придј. каже се за не-
кога ко је немаран па на њему све виси. - 
О̀на ста̏вила на̀ себе сли̏ндра̄нӯ блу́зу, а
 о̑н у̀ слиндра̄нијем га̏ћама бес каи́ша.

слѝндрат се, -а̄н се свр. објесити се
(односи се на одјећу). – Ка̏ко нѐ видӣш
да су ти се бје̏чве слѝн-драле?

сли́ндре, -е̄ ж женска особа неуредна
и немарна у облачењу. – Ко̀ја̄ је о̀но̄
сли́ндре би́ла ју̀че̄р с то̏-бо̄н у ше́тњи?

сли́ндро м неуредан и немаран човјек
кад је у питању облачење. – Ве́жи тра̏
кове на̀ цревјама, нѐ буди сли́ндро!

сли̏нит, -ӣм несвр. плакати, цмиздри-
ти. — Нѐмо̄ј са̏д повишѐ тега̄ сли̏нит, не-
го по̀купи те̑ крња́тке с по̀да.

сли̏т, сли̏је̄н свр. за кратко вријеме
попити већу количину неког пића. –
 Сли̏је̄ ли̏тру ра̏кије̄ у̀ себе ко да ли̏јева̄ у
мјѐшину.

/слобо̀дица/ изр. мо̀ја ку̏ћица, мо̀ја слобо̀д-
ица изрека која се чује у Дубровнику.

сло̀мит се, сло̏мије̄н се свр. пасти и
повриједити се. – Ако се сло̏мије̄ш са̀
скалице̄, не до̀лази ми пла̏кат!

/слу́га/ изр. ни́је (му) ни слу́га каже се
кад неко или нешто за неким или за не-
чим заостаје по ква-литету, интелиген-
цији и сл. – Њѐгово ви́но ни́је ни слу́га
тво̑му. – Нѐ може о̑н замије́нит бра̏та, ма
ни́је му ни слу́га! слу̀гаïва̄нïсе поздрав ти-
пичан за Дубровник, најчешће употре-
бљаван и то у свим ситуаци-јама: прили-
ком уласка и изласка, у сусрету на улици
итд.; значење: слуга Вам се клања или 
сл. – Слу̀гаïва̄нïсе, госпођа Ка̑те! Ка̏ко
сте? – Слу̀гаïва̄нïсе! Дра̑го ми Вас је ви̏-
ђет у̀ мене у бу̀тӣзи. Слу́жите се!

слу̀жба ж служинчад. – У нѐђељу су
го̀спа̄р и госпо̀ђа са̑ми, јер слу̀жба ѝзӣђе̄
на̀дво̄р.

слу́жит, слу̑жӣн несвр.  I. 1. бити по-
требан некоме. – Слу̑жӣ ли ти о̀ва̄ на̏пи-

ца или ћу је спре́мит? – Не́ћу ѝмат но̏ва̄-
ца̄ ко̀лико̄ ми слу̑жӣ за све̏ о̀во̄ пла́тит. 2.

 дворити некога, бити некоме слуга. —

Слу́жи-ла је, кад је би́ла мла́да, у госпо̀ђе̄
Де̏ше̄. изр. би̏ћеш слу̑жен – би̏ћете слу̑жени
одговор на поздрав којим неко инди—
ректно, односно посредно, поздравља.
– „По̀здравите ми Ва̏шу дра̑гӯ сѐстру!“
– „Фа́ла, би̏-ћете слу̑жени!“  II. ~ се 1. из-
вољети. – Е̏во ка̀фе̄! Слу́жите се, госпо-
̀ђа Лу̑кре! 2. бити стална муштерија у
некој трговини. – Слу́жили смо се два-
́дести го̏дӣшта̄ у̀ исто̄н бу̀тӣзи.

слу̏ша̄ње с гл. им. од слу̏шат послу-
шност, покоравање некоме. – О̀(д) тебе
и̏ште̄н ва̏ља̄нос и слу̏-ша̄ње.

слу̏шат, -а̄н несвр. бити послушан,
прихватати савјете, наређења, бити
покоран. – Слу̏ша̄ ли те си̑н или је по̏чео
чѝнит што̏ га во̏ја?

сљѐпица ж 1. собица без прозора. — У 
сљѐпици је таво̀лӣн су̏ два̄ сто́чића. 2.
улица која нема изла-за, ћор-сокак. — На̏-
ша у̏лица је би́ла сљѐпица и мо̀гло се у 
њо́ме сло̏бодно ѝграт.

сма̏гнут, -не̄м несвр. чезнути за нечим
што је пред очима нпр. за јелом. – То̑
дије́те ва̏зд сма̏гне̄ за̀ нечијем што̏ му нѐ
можеш пру̏жит.

сма̏зат се, -же̄ се свр. (само 3. л.) кад
се облаци згусну и небо се зацрни. – Све̏
се сма̏зало, мо̑ра̄ не̏-што учѝнит!

сма̀кнут, сма̏кне̄м свр.  I. 1. уништи-
ти. – О̀ва̄ дјѐца ће ме сма̀кнут. 2. брзо и
до краја појести не-што. – Док си о̏ком
тре̏пнуо, све̏ су сма̀кнули.  II. ~ се склиз—
нути с неког одређеног мјеста. – Сма̏—
ко̄ ми се пр̏сте̄н с ру́ке̄. – Сма̀кнуће се та̑
плѝтица о̀то̄ле и ра̀збиће се.

сма̀нтрат, -а̄н свр. ошамутити. –
Сма̀нтрала ме је велика̄ конфузѝјо̄н по
гра́ду.

сме̑с се в. смести се.

сме́сти се, -те̄н се свр. (и:  сме́стит се,
сме̑с се) кад нешто слабо напредује, нпр.
