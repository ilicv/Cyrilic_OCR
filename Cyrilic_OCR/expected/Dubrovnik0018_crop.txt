слѝнгурина ж (пеј. од слѝнгура) – 
Јѐдна му је ћи̑ у̀шесна, а дру̏га̄ пра̑ва̄
слѝнгурина.

сли̏ндра̄н, -а, -о придј. каже се за не-
кога ко је немаран па на њему све виси. - 
О̀на ста̏вила на̀ себе сли̏ндра̄нӯ блу́зу, а
 о̑н у̀ слиндра̄нијем га̏ћама бес каи́ша.

слѝндрат се, -а̄н се свр. објесити се
(односи се на одјећу). – Ка̏ко нѐ видӣш
да су ти се бје̏чве слѝндрале?

сли́ндре, -е̄ ж женска особа неуредна
и немарна у облачењу. – Ко̀ја̄ је о̀но̄
сли́ндре би́ла ју̀че̄р с то̏бо̄н у ше́тњи?

сли́ндро м неуредан и немаран човјек
кад је у питању облачење. – Ве́жи тра̏
кове на̀ цревјама, нѐ буди сли́ндро!

сли̏нит, -ӣм несвр. плакати, цмиздри-
ти. — Нѐмо̄ј са̏д повишѐ тега̄ сли̏нит, не-
го по̀купи те̑ крња́тке с по̀да.

сли̏т, сли̏је̄н свр. за кратко вријеме
попити већу количину неког пића. –
 Сли̏је̄ ли̏тру ра̏кије̄ у̀ себе ко да ли̏јева̄ у
мјѐшину.

/слобо̀дица/ изр. мо̀ја ку̏ћица, мо̀ја слобо̀д-
ица изрека која се чује у Дубровнику.

сло̀мит се, сло̏мије̄н се свр. пасти и
повриједити се. – Ако се сло̏мије̄ш са̀
скалице̄, не до̀лази ми пла̏кат!

/слу́га/ изр. ни́је (му) ни слу́га каже се
кад неко или нешто за неким или за не-
чим заостаје по квалитету, интелиген-
цији и сл. – Њѐгово ви́но ни́је ни слу́га
тво̑му. – Нѐ може о̑н замије́нит бра̏та, ма
ни́је му ни слу́га! слу̀гаïва̄нïсе поздрав ти-
пичан за Дубровник, најчешће употре-
бљаван и то у свим ситуацијама: прили-
ком уласка и изласка, у сусрету на улици
итд.; значење: слуга Вам се клања или 
сл. – Слу̀гаïва̄нïсе, госпођа Ка̑те! Ка̏ко
сте? – Слу̀гаïва̄нïсе! Дра̑го ми Вас је ви̏-
ђет у̀ мене у бу̀тӣзи. Слу́жите се!

слу̀жба ж служинчад. – У нѐђељу су
го̀спа̄р и госпо̀ђа са̑ми, јер слу̀жба ѝзӣђе̄
на̀дво̄р.

слу́жит, слу̑жӣн несвр.  I. 1. бити по-
требан некоме. – Слу̑жӣ ли ти о̀ва̄ на̏пи-

ца или ћу је спре́мит? – Не́ћу ѝмат но̏ва̄-
ца̄ ко̀лико̄ ми слу̑жӣ за све̏ о̀во̄ пла́тит. 2.

 дворити некога, бити некоме слуга. —

Слу́жила је, кад је би́ла мла́да, у госпо̀ђе̄
Де̏ше̄. изр. би̏ћеш слу̑жен – би̏ћете слу̑жени
одговор на поздрав којим неко инди—
ректно, односно посредно, поздравља.
– „По̀здравите ми Ва̏шу дра̑гӯ сѐстру!“
– „Фа́ла, би̏ћете слу̑жени!“  II. ~ се 1. из-
вољети. – Е̏во ка̀фе̄! Слу́жите се, госпо̀-
ђа Лу̑кре! 2. бити стална муштерија у
некој трговини. – Слу́жили смо се два́
дести го̏дӣшта̄ у̀ исто̄н бу̀тӣзи.

слу̏ша̄ње с гл. им. од слу̏шат послу-
шност, покоравање некоме. – О̀(д) тебе
и̏ште̄н ва̏ља̄нос и слу̏ша̄ње.

слу̏шат, -а̄н несвр. бити послушан,
прихватати савјете, наређења, бити
покоран. – Слу̏ша̄ ли те си̑н или је по̏чео
чѝнит што̏ га во̏ја?

сљѐпица ж 1. собица без прозора. — У 
сљѐпици је таво̀лӣн су̏ два̄ сто́чића. 2.
улица која нема излаза, ћор-сокак. — На̏-
ша у̏лица је би́ла сљѐпица и мо̀гло се у 
њо́ме сло̏бодно ѝграт.

сма̏гнут, -не̄м несвр. чезнути за нечим
што је пред очима нпр. за јелом. – То̑
дије́те ва̏зд сма̏гне̄ за̀ нечијем што̏ му нѐ
можеш пру̏жит.

сма̏зат се, -же̄ се свр. (само 3. л.) кад
се облаци згусну и небо се зацрни. – Све̏
се сма̏зало, мо̑ра̄ не̏што учѝнит!

сма̀кнут, сма̏кне̄м свр.  I. 1. уништи-
ти. – О̀ва̄ дјѐца ће ме сма̀кнут. 2. брзо и
до краја појести нешто. – Док си о̏ком
тре̏пнуо, све̏ су сма̀кнули.  II. ~ се скли—
знути с неког одређеног мјеста. – Сма̏—
ко̄ ми се пр̏сте̄н с ру́ке̄. – Сма̀кнуће се та̑
плѝтица о̀то̄ле и ра̀збиће се.

сма̀нтрат, -а̄н свр. ошамутити. –
Сма̀нтрала ме је велика̄ конфузѝјо̄н по
гра́ду.

сме̑с се в. смести се.

сме́сти се, -те̄н се свр. (и:  сме́стит се,
сме̑с се) кад нешто слабо напредује, нпр.
